packages/generator/src/eta-generator.ts

```
@@ -239,8 +239,22 @@ export class EtaTPEGCodeGenerator {
         this.collectUsedCombinators((expr as LabeledExpression).expression, combinators);
         break;
       case 'Quantified':
-        combinators.add('sequence'); // Often used in quantification implementation
-        this.collectUsedCombinators((expr as Quantified).expression, combinators);
+        const quantifiedExpr = expr as Quantified;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)